### ISSUE TITLE
[BUG]: PCSX2 crashed while playing Dog's Life (EU)

### ISSUE THREAD

**Issue by dezraj** [PCSX2/pcsx2#6502]
### Describe the Bug

I was playing the game normally and eventually it crashed.

This game is working not so smoothly as far as I've noticed. The title screen doesn't run at full speed, which I wouldn't expect to be the case considering my PC specs (see below). The dog skin in the game becomes all black eventually and when we switch to the dog vision mode there is some garbage in the screen before it stabilizes.

PC specs:
i5-10400 w/ 16gb RAM
GTX 1660 Ti
Win 10 x64

Thank you for your time and work.

### Reproduction Steps

I don't know exactly how I crashed it. Maybe switching back and forth to the dog vision mode could have triggered the crash?

### Expected Behavior

There are many closed issues related to the this game, so I supposed it would work without major problems. Maybe it's not the case yet?

### PCSX2 Revision

1.7.2996

### Operating System

Windows 10 (64bit)

### CPU

i5-10400

### GPU

GTX 1660 Ti

### GS Settings

![image](https://user-images.githubusercontent.com/27839354/175463340-8470ad1e-cca9-4dd6-b8cf-1fc959a033cf.png)
![image](https://user-images.githubusercontent.com/27839354/175463313-7a53bbb4-d9f2-4594-adc4-74e125e9da5d.png)
![image](https://user-images.githubusercontent.com/27839354/175463380-bcb10af3-d734-46ae-ba90-dfd1cf753dc7.png)

All other options are as default.

### Emulation Settings

![image](https://user-images.githubusercontent.com/27839354/175463483-66579dc7-4e0c-425d-a17f-c2c1b9d6c630.png)

### Logs & Dumps

[crash-2022-06-24-01-19-51-580.zip](https://github.com/PCSX2/pcsx2/files/8973194/crash-2022-06-24-01-19-51-580.zip)

**Comment by refractionpcsx2** [comment]
Although it's not responsible for the crash, but Native x4 is one of the upscales which doesn't work in this game with Jake turning black, use either 3x or 5x and it will be fine.

**Comment by dezraj** [comment]
I've just had another crash using v1.7.3028 and the 5x resolution. Now I virtually didn't do anything, just started the game and it crashed before I could start controlling the dog.

Crash dump:
[crash-2022-06-29-01-36-21-939.zip](https://github.com/PCSX2/pcsx2/files/9007277/crash-2022-06-29-01-36-21-939.zip)

**Comment by refractionpcsx2** [comment]
shouldn't be an issue any longer

**Comment by dezraj** [comment]
> shouldn't be an issue any longer

Thank you so much, @refractionpcsx2. I can no longer reproduce the crash as far as I've tested.

If you don't bother, I'd like to ask why would this game not be getting full speed in the title screen and in the save file select (dog house) screen. I've recorded a small video-clip to show you, here:
https://drive.google.com/file/d/1Zz4zDcL9QTTP9exCcyENdKZq5eTT__VI/view?usp=sharing

I ask because I have an i5-10400 with a GTX 1660 Ti, which I thought would be enough for at least 3x IR.

Anyway, thank you so much again for your work.
